lib/utils/app_flavors.dart

```diff
@@ -1,13 +1,22 @@
 import 'package:ardrive/services/config/config_service.dart';
+import 'package:ardrive/utils/logger/logger.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 
 class AppFlavors {
   Future<Flavor> getAppFlavor() async {
+    if (kIsWeb) {
+      if (const String.fromEnvironment('environment') == 'dev') {
+        return Flavor.production;
+      }
+
+      return Flavor.development;
+    }
```

Comment:
Shouldn't it be something like this?
So if it's not `dev` it will default to `prod`

```suggestion
    if (kIsWeb) {
      if (const String.fromEnvironment('environment') == 'dev') {
        return Flavor.development;
      }

      return Flavor.production;
    }
```

Reply:
b40d51c0